server/src/main/java/org/opensearch/index/store/RemoteSegmentStoreDirectory.java

```diff
@@ -434,6 +436,30 @@ public void copyFrom(Directory from, String src, IOContext context, ActionListen
         }
     }
 
+    /**
+     * Copies the segment from the remote directory to a local directory, preferably using an async mechanism using multi stream support.
+     * @param storeDirectory The directory where the segment needs to be copied to
+     * @param src The name of the segment file
```

Comment:
I suggest spelling out "source" because I'm annoying like that

Reply:
Updated.